lib/docs/storybook-static/11.7f35c06cd77f40a88403.manager.bundle.js.LICENSE.txt

```diff
@@ -0,0 +1,12 @@
+/*!
```

Comment:
Pretty much better already. Just don't forget to gitignore the build of storybook